llvm/lib/Target/AArch64/AArch64TargetTransformInfo.cpp

```diff
@@ -1004,6 +1004,13 @@ AArch64TTIImpl::getIntrinsicInstrCost(const IntrinsicCostAttributes &ICA,
     }
     break;
   }
+  case Intrinsic::experimental_vector_extract_last_active:
+    if (ST->isSVEAvailable()) {
```

Comment:
AFAIK, this is implemented with `+sme`, so could be:
```suggestion
    if (ST->isSVEorStreamingSVEAvailable()) {
```
(I know we don't auto-vectorize in streaming-mode, but I've seen some recent PRs where people have wanted pragma-based vectorization to work for streaming functions.)